README.dotnet-framework-build.md

```diff
@@ -0,0 +1,108 @@
+# Supported Windows Server 2016 Version 1709 (Fall Creators Update) amd64 tags
+
+- [`4.7.1-1-windowsservercore-1709`, `4.7.1-windowsservercore-1709` (*4.7.1-windowsservercore-1709/build/Dockerfile*)](https://github.com/Microsoft/dotnet-framework-docker/blob/master/4.7.1-windowsservercore-1709/build/Dockerfile)
```

Comment:
I'm assuming for the time being, you've just copied the README as is?
All these tags are missing the '-build'

Reply:
No the readme wasn't just copied.  A fair bit of it is the same but the sample and tags are different amongst other minor parts.

The `build` suffix is not necessary as the plan is for a new `microsoft/dotnet-framework-build` repo.

Reply:
Ahh. My bad.
1 GH repo builds 2 DH repos?

Reply:
correct